### ISSUE TITLE
OPTIMIZER: Update haul cost calculations for biomasscd 4

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#360]
We support biomasscd 4 in the ui and database but it is excluded from the haul cost calculations. Developing the new logic for this biomasscd is on hold pending the conversion of BioSum to SQLite

**Comment by druepdx** [comment]
Do we want to apply escalators to the wood values for the new biomass codes? If so, do we need to add new ones or do we want to use the existing merch one?

**Comment by druepdx** [comment]
Use Wood4, Wood5, Wood6 labels. Use merch escalators

**Comment by lbross** [comment]
We want to add Wood4 Volume, Wood5 Volume, and Wood6 Volume as default calculated variable options with a value of 1 for each time interval.

**Comment by druepdx** [comment]
For Optimization variables, do we want to add Wood4 Volume, Wood5 Volume, Wood6 Volume, and Non-Residue Volume as options on the Optimization screen (see below), or is having them as Economic attribute options sufficient? 

<img width="717" height="332" alt="Image" src="https://github.com/user-attachments/assets/21e861ba-e31d-4327-b0e6-708a57b852e9" />

**Comment by jsfried** [comment]
I am not sure that I understand the question. One can generated multiple economic attributes in the weighting system, right? So could have wood4_total, or Wood3_cycle1 for example, and they they would appear here as variables? Or you could bring one of them in as a variable when you clicked on Economic Attribute? If so, then I think having them handled as economic variables works fine

**Comment by jsfried** [comment]
Entering a test record here to test my email routing of notifications @jsfried

**Comment by druepdx** [comment]
Yes, the analyst can generate multiple variables in the Weighted Variables screen. For what it's worth, I agree that it makes the most sense to have them as options when selecting Economic Attribute in the screen above (like chip_volume_1, net_revenue_1, etc.) as opposed to having them all listed on the screen

**Comment by druepdx** [comment]
Updated default optimizer_definitions.db

[optimizer_definitions.zip](https://github.com/user-attachments/files/30721948/optimizer_definitions.zip)

**Comment by lbross** [comment]
Just double checking that the reason for this are the new economic variables for new wood types? Can you add a deployment section to the design spec? The first item on the list will be to deploy the new optimizer.definitions.db with the .msi.

**Comment by druepdx** [comment]
Yes, this is because of the new wood type variables and the dry weight variables

**Comment by druepdx** [comment]
New optimizer_definitions with total dry weight variable: 

[optimizer_definitions.zip](https://github.com/user-attachments/files/31195897/optimizer_definitions.zip)
